import-service/src/functions/importFileParser/handler.ts

```diff
@@ -0,0 +1,53 @@
+import * as AWS from 'aws-sdk';
+import { S3Event } from 'aws-lambda';
+import csvParser from 'csv-parser';
+
+const importFileParser = async (event: S3Event) => {
```

Comment:
Code inside of importFileParser is so hard to read, maybe it's better to split it and move to some utils to get more readable code.